proto/pdpb.proto

```diff
@@ -68,10 +68,10 @@ message AllocIdResponse {
 }
 
 message GetMetaRequest {
-    optional MetaType meta_type    = 1;
-    optional uint64 store_id       = 2;
-    optional bytes region_key      = 3;
-    optional uint64 cluster_id     = 4;
+	optional MetaType meta_type    = 1;
+	optional uint64 store_id       = 2;
+	optional bytes region_key      = 3;
+	optional string cluster_name   = 4;
```

Comment:
Seems now we can split meta into cluster config, store and region three different requests, that will be clearer.